app/components/ChallengesMapView.js

```diff
@@ -71,17 +70,17 @@ export default class ChallengesMapView extends Component {
 				<View style={{flex:1}}>
 					<MapView
 						style={styles.map}
-						loadingEnabled={true}
-						showUserLocation={true}
+						loadingEnabled={false}
```

Comment:
@jwt1143 I couldn't get this to work with it set to true. It just sat there and spun forever. I'll talk to yoojeong, but I tried it on my phone and the emulator and couldn't get it working.